Packs/JamfProtect/Integrations/JamfProtectEventCollector/JamfProtectEventCollector.py

```diff
@@ -334,6 +450,50 @@ def get_events_alert_type(client: Client, start_date: str, max_fetch: int, last_
     return events, new_last_run_without_next_page
 
 
+def get_events_computer_type(client: Client, start_date: str, max_fetch: int, last_run: dict) -> tuple[list[dict], dict]:
+    """
+    Fetches computer type events from the Jamf Protect API within a specified date range.
+
+    This function fetches computer type events from the Jamf Protect API based on the provided start date.
+    It fetches events up to the maximum number specified by max_fetch.
+    The function also uses the information from the last run to continue fetching from where it left off in the previous run.
+
+    Args:
+        client (Client): An instance of the Client class for interacting with the API.
+        start_date (str): The start date for fetching events in '%Y-%m-%dT%H:%M:%SZ' format.
+        max_fetch (int): The maximum number of events to fetch.
+        last_run (dict): A dictionary containing information about the last run.
+
+    Returns:
+        tuple: A tuple containing two elements:
+            - A list of dictionaries. Each dictionary represents an event.
+            - A dictionary with new last run values,
+             the end date of the fetched events and a continuance token if the fetched reached the max limit.
+    """
+    created, current_date = calculate_fetch_dates(start_date, last_run=last_run, last_run_key="computer")
+    command_args = {"created": created}
+    client_event_type_func = client.get_computer
+    next_page = last_run.get("computer", {}).get("next_page", "")
+
+    demisto.debug(f"Fetching computers since {created}")
+    events, next_page = get_events(command_args, client_event_type_func, max_fetch, next_page)
+    for event in events:
+        event["source_log_type"] = "computers"
+    if next_page:
+        demisto.debug(
+            f"Fetched the maximal number of computers ({len(events)} events). "
+            f"Fetching will continue using {next_page=} and last_fetch={created}, in the next iteration")
+        new_last_run_with_next_page = {"next_page": next_page, "last_fetch": created}
+        return events, new_last_run_with_next_page
+
+    new_last_fetch_date = max(filter(None, (arg_to_datetime(event.get("created"), DATE_FORMAT)
+                                            for event in events))).strftime(DATE_FORMAT) if events else current_date
+    # If there is no next page, the last fetch date will be the max end date of the fetched events.
+    new_last_run_without_next_page = {"last_fetch": new_last_fetch_date}
+    demisto.debug(f"Jamf Protect- Fetched {len(events)} computers")
```

Comment:
Integration name and instance are prepended automatically
```suggestion
    demisto.debug(f"Fetched {len(events)} computers")
```